model.py

```
@@ -1,51 +1,99 @@
+            nn.Dropout(dropout_rate),
+            nn.Linear(hidden_sizes[-1] // 2, output_size)
+        )
+
+    def forward(self, x, edge_index, batch):
+        x = self.input_proj(x)
+
+        for layer, norm, typ in zip(self.layers, self.layer_norms, self.layer_types):
+            if typ == "gcn":
+                x = F.leaky_relu(norm(layer(x, edge_index)), 0.2)
+            elif typ == "gat":
+                x = F.leaky_relu(norm(layer(x, edge_index)), 0.2)
```

Ruff (0.8.2)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)